assignment/index.html

```diff
@@ -63,9 +63,25 @@
       ===================== */
 
       var jsonToCsv = function(json) { console.log(json); };
+      var addMarkers = function(map) {};
 
 
-      var addMarkers = function(map) {};
+//Part 1
+//Log a series of arrays to the console that represents the health_centers dataset in array form.
+//Code guidance: https://openclassrooms.com/en/courses/3523231-learn-to-code-with-javascript/3703666-store-data-in-arrays
+
+for (var i = 0; i < healthCenters.length; i=i+1) {//sets loop through each element starting at 0 (i=0) for full length of array minus 1 (last value)
+    console.log(healthCenters[i]);//logs to console set of health centers in array form
```

Comment:
This does not produce an array of arrays. 

Your results console log each object as such:

![image](https://user-images.githubusercontent.com/8103418/52538633-fd2a2580-2d42-11e9-8b96-ecea8c0bb28c.png)

We are looking for you to produce an array of arrays as such:

![image](https://user-images.githubusercontent.com/8103418/52538636-087d5100-2d43-11e9-9f7d-8360e6d1b1c4.png)

Reply:
I just added a revision that I believe addresses the issue.  I initially did not see that the push worked so I pushed it twice...second only has a change in spacing.
![image](https://user-images.githubusercontent.com/46828136/54314078-14159f00-45b1-11e9-9eee-25ded0a6cc9a.png)